### PR TITLE
Create note deletion page UI

### LINKED ISSUE #2: Create note deletion page UI

### PR DESCRIPTION
**Pull request by abrahamparayil**
Fixes: #2

- a9e60df Created note deletion page UI

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -30,9 +30,7 @@ const DeleteAlert = ({
       onSubmit={handleDelete}
       onClose={onClose}
       message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      title={`Delete note?`}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:33]: Here, I guess we are using neetoUI Modal. Can we switch to neetoUI alert?

**Review by karthiknmenon — COMMENTED** [review]
@abrahamparayil A minor suggestion. Please check.

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx (hunk 1 above)
Comment (abrahamparayil) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:33]: I went with what was being used in wheel by default. Will change.

Comment (abrahamparayil) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:33]: Actually, neetoUI Alert is being used here.
Reference: https://github.com/abrahamparayil/wheel/blob/a9e60dfe7198e547cbcdc5e75015c1b6b9e47558/app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx#L3

**Pull request merged**